AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -461,7 +459,7 @@ - (void)dealloc
 
 - (void)__unloadNode
 {
-  ASDisplayNodeAssertThreadAffinity(self);
```

Comment:
It always has been, since we deallocate the view here, and since we assert thread affinity and also assert node is loaded. This just makes it clearer and I hate the phrase "ThreadAffinity."